src/esm_runscripts/filedicts.py

```diff
@@ -193,7 +225,7 @@ def cp(self, source: str, target: str) -> None:
 
     @_allowed_to_be_missing
     def ln(self, source: AnyStr, target: AnyStr) -> None:
-        """creates symbolic links from the path retrieved by ``source`` to the one by ``target``
+        """creates symbolic links from the path retrieved by ``source`` to the one by ``target``. ``source`` and ``target`` keys will be transformed into their corresponding absolute paths and ``target`` will be created.
```

Comment:
I don't understand the extra documentation here. Nothing gets transformed in this function right?